npm_and_yarn/lib/dependabot/npm_and_yarn/helpers.rb

```diff
@@ -24,23 +24,48 @@ def self.fetch_yarnrc_yml_value(key, default_value)
         end
       end
 
+      def self.yarn_berry?
+        yarn_major_version >= 2
+      end
+
+      def self.yarn_major_version
```

Comment:
Should we memoize this? It's not hugely expensive, but we are shelling out and the value would never change I think?

Reply:
I need to fix this as it is the cause of the updater CI failure. I think we have to determine the version based on a lockfile as we were previously.